src/display.js

```diff
@@ -0,0 +1,199 @@
+import OneTask from './one_task.js';
+import tasks from './tasks.js';
+
+const root = document.getElementById('parent');
+
+class Display {
+  createElement = (type, cls, id) => {
+    const element = document.createElement(type);
+
+    if (cls) {
+      for (let i = 0; i < cls.length; i += 1) {
+        element.classList.add(cls[i]);
+      }
+    }
+    if (id) {
+      element.id = id;
+    }
+    return element;
+  };
+
+  appendElement = (child, parent) => {
+    parent.appendChild(child);
+  };
```

Comment:
The createElemet function you've used is a nice approach to implement the DRY principle in your code.